src/app/api/catalogs/models.ts

```
@@ -115,7 +111,47 @@ function createPlaylistId(channel: string) {
+        channelLogo: playlist.channelLogo,
+        channelTitle: item.snippet.channelTitle,
+        videoId: item.contentDetails.videoId,
+        description: item.snippet.description,
+        publishedAt: item.contentDetails.videoPublishedAt,
+      });
+    }
+  } catch (err) {
+    console.error(err);
+  } finally {
+    return playlistItemData;
+  }
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)